frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -74,12 +86,31 @@ export function Customization({
                     {validationErrors?.borderColor && <LemonField.Error error={validationErrors?.borderColor} />}
                 </LemonField.Pure>
                 <>
-                    <LemonField.Pure className="mt-2" label="Position">
+                    <LemonField.Pure
+                        className="mt-2"
+                        label="Position"
+                        info={
+                            isWidgetSurveyWithSelectorWidgetType
+                                ? 'The "next to trigger" option requires posthog.js version TODO_BEFORE_MERGE or higher.'
```

Comment:
is 'next to trigger' obvious for a reader? if they set a selector, should we say something like next to the chosen widget's (or component) selector or something?

Reply:
changed to `This option is only available for feedback button surveys. The survey will be displayed next to the chosen feedback button, based on the CSS selector you provided`